modules/cnf-managing-device-interrupt-processing-for-guaranteed-pod-isolated-cpus.adoc

```diff
@@ -0,0 +1,58 @@
+// CNF-802 Infrastructure-provided interrupt processing for guaranteed pod CPUs
+// Module included in the following assemblies:
+//
+// *cnf-performance-addon-operator-for-low-latency-nodes.adoc
+
+[id="managing-device-interrupt-processing-for-guaranteed-pod-isolated-cpus_{context}"]
+= Managing device interrupt processing for guaranteed pod isolated CPUs
+
+The Performance Addon Operator manages host CPUs by dividing them into reserved CPUs for cluster housekeeping duties, and isolated CPUs for workloads. CPUs that are used for real-time workloads are set as isolated. Being able to configure device interrupt processing means that administrators can balance the number of reserved CPUs against the number of guaranteed pod CPUs for low-latency processing. In the performance profile, `GloballyDisableIrqLoadBalancing` is used to manage whether device interrupts are processed or not.
+
+To achieve real-time latency for workloads, some (but not all) pods require the CPUs they are running on to not process device interrupts.
+
+For certain workloads the reserved CPUs are not always sufficient for dealing with device interrupts, and for this reason, device interrupts are not globally disabled on the isolated CPUs. By default, PAO does not disable device interrupts on isolated CPUs. A pod annotation, `irq-load-balancing.crio.io`, is used to define whether device interrupts are processed or not.
```

Comment:
I would use the whole name "Performance Addon Operator" with link to the github docs if is allowed

Reply:
@marcel-apf which gh link are you thinking about specfically? is it: https://github.com/openshift-kni/performance-addon-operators/blob/master/docs/performance_profile.md